src/client/styles/main.scss

```diff
@@ -54,6 +54,10 @@ button {
   @include box-sizing(border-box);
 }
 
+.hidden {
```

Comment:
si... there is another one called .nypl-screenreader-only which is *not* the same as .hidden but, i suppose, the same as .visuallyHidden

i do need to add the .hidden crutch for now, though... (and i have it in one of the PRs)